migrations/Version202109201454385081_taoClientDiagnostic.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
```

Comment:
Should we add the license text block here?

Reply:
Good question, usually we don't for the migration scripts, but maybe is it a bad habit. I'll add them.